Cherry pick open modal listing scenes owned by me (issue #222)

kfarr opened the pull request:
to merge:
- [x] Zade - styling issues - see below
- [x] Zade - remove search, "load new scene" and "create new scene" buttons for now (can just comment out, we can use later)
- [x] Zade - when clicking a scene, the scene should load and the dialog should close and the scene should be loaded
- [x] Zade - issue regarding loading new items when modal is loaded, instead of once upon authentication. (In other words, if the application is open, and you use firebase to change a title, for instance, the new scene title is not reflected in the list of scenes.)

Styling issues:
* Use hover states for cards: https://www.figma.com/file/W9jid3A0jgssnIKBFEvG38/3DStreet-2022?type=design&node-id=3131-48379&mode=design&t=y8oGhampc6fY6mWb-4
* Hover cursor for cards should be same as an `<a>` link a hand pointer
* Modal title "Open scene" is not aligned correctly, see https://www.figma.com/file/W9jid3A0jgssnIKBFEvG38/3DStreet-2022?type=design&node-id=5012-124637&mode=design&t=y8oGhampc6fY6mWb-4
* Also too much margin on the left, could be cut in half
* Cards should wrap text -- TBD
* Scrollbar should be visible per figma: https://www.figma.com/file/W9jid3A0jgssnIKBFEvG38/3DStreet-2022?type=design&node-id=5012-124637&mode=design&t=y8oGhampc6fY6mWb-4 (reduce screen size to see scrollbar)
* Visual notes: 
![image](https://github.com/3DStreet/3dstreet-editor/assets/470477/4f10887b-3a59-41f7-beae-37cc3eb1dadc)

kfarr committed:
feat: scenes modal

Co-Authored-By: Rostyslav Nahornyi <76824095+rostyslavnahornyi@users.noreply.github.com>

kfarr committed:
refactor: scenes modal

Co-Authored-By: Rostyslav Nahornyi <76824095+rostyslavnahornyi@users.noreply.github.com>

kfarr committed:
Merge branch 'master' into cherry-pick-open-modal-listing-scenes-owned-by-me-issue-222

rostyslavvnahornyi committed:
feat: opening scenes owned by user

rostyslavvnahornyi commented:
Hello @kfarr! 
I pushed the changes to the `cherry-pick-open-modal-listing-scenes-owned-by-me-issue-222` branch. Can you check the last commit here pls.

Will be very grateful)

rostyslavvnahornyi committed:
Merge branch 'feature-firebase-save-load' into cherry-pick-open-modal-listing-scenes-owned-by-me-issue-222

kfarr commented:
Almost there!
- [x] (DONE) opening scene should always update the URL hash when opening scene (as a user, when I copy the URL path and re-open it should return me to the scene that I was viewing). So if I click on a scene with UUID `abcd-1234` running on host `dev-3dstreet.web.app` then the hash should change to `#/scenes/abcd-1234.json` therefore the full URL bar should be `https://dev-3dstreet.web.app/#/scenes/abcd-1234.json`. An example of this is here: https://github.com/3DStreet/3dstreet-editor/blob/feature-firebase-save-load/src/components/scenegraph/Toolbar.js#L142
- [x] wrapping text: for scenes with long titles, they should show ellipses (`...` dots) if they are too long, and then show full name on cursor hover as tooltip

kfarr committed:
add scene hash when opening from modal

kfarr committed:
add load scene notification

kfarr committed:
style changes

- fixed card size regardless of image used, space for second line for date per design
- card hover shows full title
- modal width to 80vw instead of fixed
